packages/blinks/src/index.css

```diff
@@ -6,6 +6,11 @@
   .blink {
     container-type: inline-size;
   }
+  /* Restricting the Width only for Youtube - can be done either ways */
+  /* Can also change depending on the platform */
+  .blink.youtube-dark, .blink.youtube-light {
```

Comment:
I'd recommend controlling the width through `youtube.tsx`, where the injected Blink component is rendered with a max-w wrapper

Reply:
Done, Please check the below Commit : 6e02c65a115ac050bb3224f0c3bb99aa85c637a4